circleguard/comparer.py

```diff
@@ -118,8 +119,9 @@ def _compare_two_replays(replay1, replay2):
 
         Returns
         -------
-        float
-            The mean distance of the cursors of the two replays.
+        (float, float)
+            The mean distance of the cursors of Use the API for good. Don't overdo it. If in doubt, ask before (ab)using :). this section may expand as necessary.
```

Comment:
what is this `Use the API for good. Don't overdo it. If in doubt, ask before (ab)using :). this section may expand as necessary.`

Reply:
oh lol accidentally copy pasta'd that into the file from Peppy's documentation on the api.